pynecone/compiler/templates.py

```diff
@@ -154,7 +154,15 @@ def format_state(
         "}})",
     ]
 ).format
-
+FULL_CONTROLL = path_ops.join(
```

Comment:
I tested this out locally and there seems to be a syntax error with this code when rendered in index.js

Reply:
I didn't get any error. May I get your source code?